src/components/InfoBar/index.tsx

```diff
@@ -50,22 +50,22 @@ export default function InfoBar(props: Props) {
     <div>
       <AppBarFiller />
       <Title>Status</Title>
-      <Data>{info?.connectivityStatus}</Data>
+      <Data>{info.data?.connectivityStatus}</Data>
 
       <Title>xDai balance</Title>
-      <Data>{balances?.native && utils.formatEther(balances.native)}</Data>
+      <Data>{balances.native && utils.formatEther(balances.native)}</Data>
 
       <Title>mHOPR balance</Title>
-      <Data>{balances?.hopr && utils.formatEther(balances.hopr)}</Data>
+      <Data>{balances.hopr && utils.formatEther(balances.hopr)}</Data>
 
       <Title>Peers seen in the network</Title>
-      <Data>{peers?.announced?.length || '-'}</Data>
+      <Data>{peers.data?.announced?.length || '-'}</Data>
```

Comment:
anytime `data` is used in the fe we can assume that it will re render whenever isFetching changes so lets try to keep .data out of the frontend